test/index.js

```diff
@@ -1,5 +1,62 @@
 const tap = require('tap');
 
-tap.test('Error case', test => {
-    test.done();
-});
+const headLinesJson = require('./fixtures/getHeadlines.json');
+const opinionJson = require('./fixtures/getOpinionOn.json');
+const reviewJson = require('./fixtures/getReview.json');
+
+var lambda = require('../src/index').handler;
+
+tap.test('Test get headlines intent', test => {
+	test.plan(2);
```

Comment:
Declaring how many tests we're going to execute in the block. This is needed because when we run async tests we want to make sure that if a test doesn't return is considered as a failed test.